modal_functions/sravaani_asr.py

```
@@ -0,0 +1,137 @@
+            # Clean up temporary file
+            if os.path.exists(audio_path):
+                os.remove(audio_path)
+
+
+def download_audio(url: str) -> str:
+    """Download WAV audio file from URL to a temporary file."""
+    import requests
+    import tempfile
+    import os
+
+    response = requests.get(url, stream=True)
```

Ruff (0.16.1)

[error] Probable use of `requests` call without timeout

(S113)